storage/migrations/01_oasis_3_consensus.up.sql

```diff
@@ -251,6 +251,30 @@ CREATE TABLE oasis_3.votes
   PRIMARY KEY (proposal, voter)
 );
 
+-- Related Accounts Data
+
+CREATE TABLE oasis_3.accounts_related_transactions
+(
+  account_address oasis_addr NOT NULL,
+  tx_block UINT63 NOT NULL,
+  tx_index UINT31 NOT NULL,
+  FOREIGN KEY (tx_block, tx_index) REFERENCES oasis_3.transactions(block, tx_index) DEFERRABLE INITIALLY DEFERRED
+);
+CREATE INDEX ix_accounts_related_transactions_address_block_index ON oasis_3.accounts_related_transactions (account_address, tx_block, tx_index);
```

Comment:
I think we want two separate indices. One over `(tx_block, tx_index)` and one over `account_address`. The former will help when filtering by `rel`, the latter will make the join with the `transactions` table efficient.

Reply:
Got it, will add!

Reply:
Actually ... since we're only using this in a single type of query ("get me txs that have a related account X"), even just the `account_address` index should suffice. The planner should be smart enough to handle `rel` queries by first filtering on this table (with the index), then using the known `(tx_block, tx_index)` and the db index on the `transations` table to pull up the relevant tx rows.

Reply:
Looks like you forgot to update.
```suggestion
CREATE INDEX ix_accounts_related_transactions_address_block_index ON oasis_3.accounts_related_transactions (account_address);
```